### PR TITLE
feat: add support for maxLatency

### PR DESCRIPTION
**Pull request by cctdaniel**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -285,10 +286,12 @@ export const parsePriceData = (data: Buffer, currentSlot?: number): PriceData =>
   const timestamp = readBigInt64LE(data, 96)
   // minimum number of publishers for status to be TRADING
   const minPublishers = data.readUInt8(104)
+  // message sent flag
+  const messageSent = data.readUInt8(105)
```
Comment (cctdaniel) [inline src/index.ts:290]: drive-by: previously added to the oracle program but js client wasn't updated

File: src/index.ts [hunk 2]
```diff
@@ -394,5 +398,5 @@ export const parsePermissionData = (data: Buffer): PermissionData => {
 
 export { PythConnection } from './PythConnection'
 export { PythHttpClient } from './PythHttpClient'
-export { getPythProgramKeyForCluster, PythCluster, getPythClusterApiUrl } from './cluster'
-export { pythOracleProgram, pythOracleCoder, pythIdl } from './anchor'
+export { pythIdl, pythOracleCoder, pythOracleProgram } from './anchor'
+export { PythCluster, getPythClusterApiUrl, getPythProgramKeyForCluster } from './cluster'
```
Comment (cctdaniel) [inline src/index.ts:402]: drive-by: format

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 2.w0.0
```
Comment (guibescos) [inline CHANGELOG.md:3]: typo

**Review by guibescos — APPROVED** [review]
Good, you will also need to update the idl to support the new instruction SetMaxLatency, so maybe worth doing that before making a release

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pythnetwork/client",
-  "version": "2.19.0",
+  "version": "2.20.0",
```
Comment (ali-bahjati) [inline package.json:3]: please update the lockfile as well.

**Review by guibescos — APPROVED** [review]
